### PR TITLE
docs(quickstart): add stonedb-8.0 compiling and running guide #1449

### LINKED ISSUE #1449: docs: add stonedb-8.0 compiling and running guide
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/stoneatom/stonedb/blob/stonedb-5.7-dev/CONTRIBUTING.md).

### Description

Describe the new method to compile and run stonedb-8.0 in developer guide.

### Self-service

- [X] I'd be willing to address this documentation request myself.

### PR DESCRIPTION
**Pull request by Double0101**
## Summary about this PR

Issue Number: close #1449

## Tests Check List

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No code

## Changelog

- [ ] New Feature
- [ ] Bug Fix
- [ ] Performance Improvement
- [ ] Build/Testing/CI/CD
- [x] Documentation
- [ ] Not for changelog (changelog entry is not required)

## Documentation

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features

### CODE REVIEW

File: compile-using-ubuntu2204-for-80.md [hunk 1]
```diff
@@ -0,0 +1,307 @@
+# Compile StoneDB for MySQL8.0 on Ubuntu 22.04
+
+This topic describes how to compile StoneDB for MySQL8.0 on Ubuntu 22.04.
+
+## Precautions
```
Comment (hustjieke) [inline compile-using-ubuntu2204-for-80.md:5]: leave a blank line here.

File: compile-using-ubuntu2204-for-80.md [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+Ensure that the tools and third-party libraries used in your environment meet the following version requirements:
+* GCC 9.4.0
+* Make 3.82 or later
+* CMake 3.7.2 or later
+* marisa 0.77
+* RocksDB 6.12.6
+* Boost 1.77
+
+## Procedure
```
Comment (hustjieke) [inline compile-using-ubuntu2204-for-80.md:15]: leave a blank line here.

File: compile-using-ubuntu2204-for-80.md [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+### Step 1. Install the dependencies
```
Comment (hustjieke) [inline compile-using-ubuntu2204-for-80.md:17]: leave a blank line here.

**Review by hustjieke — APPROVED** [review]
LGTM

File: compile-using-ubuntu2204-for-80.md
Comment (Nliver) [inline compile-using-ubuntu2204-for-80.md:1]: Thank you for your contribution to the document, however, you should carefully understand the format and contents of the document before contributing

File: compile-using-ubuntu2204-for-80.md [hunk 4]
```diff
@@ -0,0 +1,338 @@
+# Compile StoneDB for MySQL8.0 on Ubuntu 22.04
```
Comment (Nliver) [inline compile-using-ubuntu2204-for-80.md:1]: The beginning of this document file is missing the related parameters of the official website document configuration. Please check the beginning of the document in the same directory.
```suggestion
# Compile StoneDB for MySQL8.0 on Ubuntu 22.04
```

**Review by Nliver — CHANGES_REQUESTED** [review]

**Pull request closed without merging**